science/marx/Portfile

```diff
@@ -22,17 +22,14 @@ license             GPL-2
 homepage            http://space.mit.edu/cxc/marx/
 master_sites        ftp://space.mit.edu/pub/cxc/marx/v5.3
 
-checksums           rmd160  658fd94bb01afe91ba843a03f56e1ca1a9993faf \
-                    sha256  24aa3734740336f05fef6eb52e1e543965e695e88e148345423f547bcce98b61
+checksums           rmd160  c117e0d48256b08a59e89c9aac61f2cd84a050ce \
+                    sha256  60b8c2a17f3e37910eef48deed4e8d3fd7b51b21748c7e930048298a011f168f
 
 distname            marx-dist-${version}
 worksrcdir          marx-${version}
 
-compiler.whitelist  macports-gcc-4.9 macports-gcc-4.8 macports-gcc-4.7 \
-                    macports-gcc-4.6 macports-gcc-4.5 macports-gcc-4.4
-compiler.fallback   macports-dragonegg-3.5-gcc-4.8 \
-                    macports-dragonegg-3.5-gcc-4.7 \
-                    macports-dragonegg-3.5-gcc-4.6 \
-                    macports-dragonegg-3.5-gcc-4.5
-
```

Comment:
Is it really the case that _any_ compiler will now work, or do we need a new whitelist?

Reply:
It works with clang/llvm and gcc. Previous whitelist was meant to ensure gcc (in fact, I only made this a macport, because users frequently run into problems when trying to compile on Macs with default clang compiler).
Not tested on Intel-C, Borland group etc. but my understanding is that macports uses Apple's clang/llvm by default anyway.